packages/swing-store/test/deletion.test.js

```diff
@@ -170,7 +172,18 @@ const setupTranscript = async (t, keepTranscripts) => {
   };
   const [dbDir, cleanup] = await tmpDir('testdb');
   t.teardown(cleanup);
-  const store = initSwingStore(dbDir, { exportCallback, keepTranscripts });
+  const [archiveDir, cleanupArchives] = await tmpDir('archives');
+  t.teardown(cleanupArchives);
+  const archiveTranscript = makeArchiveTranscript(archiveDir, {
+    fs,
+    path,
+    tmp,
+  });
+  const store = initSwingStore(dbDir, {
+    exportCallback,
+    keepTranscripts,
+    archiveTranscript,
```

Comment:
Given that the swingStore does extra / different work when `archiveTranscript` is present, should this be a variation of the test?

Same for `archiveSnapshot`, especially if we end up teeing the compression stream.

Reply:
The work is purely additive, so I figured it was fine to always check in these tests rather than multiply its configuration to the cross product of keepTranscripts={true,false} and archiveTranscript={undefined,fn}—plus there's lots of testing that covers `initSwingStore` with archiveTranscript=undefined, including in this file. Do you feel strongly about making this a test variation?

Reply:
Not strongly no, but since we already have a macro, I thought it could be cheap enough. Not all permutations need to be tested (e.g. transcript options are fully independent of the snapshot option)